src/OceanSeaIceModels/CrossRealmFluxes/atmosphere_ocean_fluxes.jl

```diff
@@ -1,11 +1,17 @@
+# Fallback
+@inline extrinsic_vector(i, j, k, grid, uₒ, vₒ) = uₒ, vₒ
+@inline intrinsic_vector(i, j, k, grid, uₒ, vₒ) = uₒ, vₒ
```

Comment:
We don't need these anymore now that they are in Oceanigans
```suggestion
@inline extrinsic_vector(i, j, k, grid, uₒ, vₒ) = uₒ, vₒ
@inline intrinsic_vector(i, j, k, grid, uₒ, vₒ) = uₒ, vₒ
```